src/components/Homepage/OurWork.js

```diff
@@ -109,22 +127,20 @@ const CaseStudy = ({ caseStudy }) => {
 
   return (
     <AnimatedLink to={`/case-study/${slug}`} title={title}>
-      <Card
-        width={100}
-        height={130}
-        color={posterColor}
-        image={alternativePreviewImage}
-      >
+      <Card width={100} height={130} color={posterColor}>
         <CardHeader backgroundColor={posterColor}>
           <section>
-            <BodyPrimary reverse={reverseColor} muted>
+            <BodyPrimary noPaddingTop reverse={reverseColor} muted>
```

Comment:
Using this as boolean attributes causes React warnings :/
Do we have any plans to improve the modifier handling to keep it easy to use but also avoid warnings

Reply:
as per your slack comment we could probably refactor to remove the styled-is?